CMakeLists.txt

```diff
@@ -100,6 +101,12 @@ if (BUILD_SHARED_LIBS)
 	message(STATUS "Shared library soversion: ${FALCOSECURITY_SHARED_LIBS_SOVERSION}")
 endif()
 
+if(NOT WIN32 AND NOT APPLE)
+	option(NO_LINK_GRPC_LIBS "Do not link gRPC libraries" OFF)
+endif()
```

Comment:
These 3 lines can easily be moved to collector's repo, since we don't support Windows or MacOS, we should be able to just set `NO_LINK_GRPC_LIBS` to `OFF`.

Reply:
this is actually already forced to OFF in `collector/CMakeLists.txt` so I'll remove this.